src/layouts/index.tsx

```diff
@@ -11,9 +10,7 @@ const Layout: React.FC<LayoutProps> = ({ children }) => {
   return (
     <>
       <Flex as="header" width="full" align="center">
-        <Box marginLeft="auto" marginTop={5} marginRight={5}>
-          <ThemeToggle />
-        </Box>
+        <Box marginLeft="auto" marginTop={5} marginRight={5}></Box>
```

Comment:
Do we still need this Box and even Flex components after the theme toggle button has been removed?

Reply:
I will deleted that one